CSS-Variables-and-JS/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Scoped CSS Variables and JS</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+   <div class="wrapper">
+    <div class="controls">
+        <h2>Фотофильтры</h2>
+<div class="filters">
```

Comment:
Code should be alway clear and readable. Don't forget about formatting.

Reply:
Ok